common/src/main/java/org/conscrypt/Conscrypt.java

```diff
@@ -86,6 +87,9 @@ private Version(int major, int minor, int patch) {
                 patch = Integer.parseInt(props.getProperty("org.conscrypt.version.patch", "-1"));
             }
         } catch (IOException e) {
+        } finally {
+            if (stream != null)
+                stream.close();
```

Comment:
Please address build failure:

common/src/main/java/org/conscrypt/Conscrypt.java#L92
error: unreported exception IOException; must be caught or declared to be thrown